.github/workflows/ci.yml

```diff
@@ -342,5 +384,5 @@ jobs:
       uses: re-actors/alls-green@release/v1
       with:
         # "allowed-skips" lists jobs that are optional but should not fail
-        allowed-skips: browser-tests-components, deploy-docs, deploy-storybook, browser-tests-docs
+        allowed-skips: browser-tests-components, deploy-docs, deploy-storybook, browser-tests-docs, build-ubuntu-node-20, unit-tests
```

Comment:
This PR highlighted a bug in the existing setup - this change doesn't trigger a build or unit tests which meant CI failed. Allowing these two steps to be skipped (but not fail) is the desired behaviour